modules/@apostrophecms/schema/ui/apos/components/AposInputCheckboxes.vue

```diff
@@ -69,6 +78,21 @@ export default {
       }
 
       return false;
+    },
+    selectItems(choice) {
+      if (choice.value === 'all') {
```

Comment:
prepend `__` to that special value so it is unlikely to conflict with a reasonable value for a regular choice.

Reply:
Nice observation, I fixed that.